### ISSUE TITLE
Plurals in apposition derive invalid inferences

### ISSUE THREAD

**Issue by assaft** [wenkokke/pelican#30]
There's a problem with the way we model plurals in appositives. It allows to derive invalid inferences such as:
T: [[[The doctors] [APP [[John and] Mary]]] [received [the [Novel Prize]]]]
H: [John [is Mary]]

Annotations (ignore the ids):
Text:
1 - The - THE
2 - doctors - N
25 - APP - WHO_A
4 - John - NP_D
5 - and - AND
6 - Mary - NP_D
8 - received - V_2
9 - the - THE
10 - Novel - MR
11 - Prize - N

Hypothesis:
[John [is Mary]]
1 - John - NP_D
2 - is - IS
3 - Mary - NP_D

The generated predcalc is:

formulas(assumptions).
% Pragmatics:
all x0 ((prize(x0) & novel_prize(x0)) <-> x0=c1).
all x0 (doctors(x0) <-> x0=c3).
all x0 ((x0=c3 & (x0=John & x0=Mary)) <-> x0=c2).

% Semantics:
received(c1, c2).

end_of_list.

formulas(goals).
Mary=John.
end_of_list.

In principle there are two problematic statements here. 
1. all x0 (doctors(x0) <-> x0=c3).
This is problematic because c3 is a constant entity while 'doctors' might be true of several different entities.
1. all x0 ((x0=c3 & (x0=John & x0=Mary)) <-> x0=c2).
   This is a totally wrong representation, and due to that the (naturally invalid) hypothesis follows. 

The bottom line is that we prove things that are not supposed to be proven. This makes the proof-system less reliable and less sound. So it's a critical issue. However, I don't see an easy way to solve this issue. The denotation of "AND" is for distributive readings, suitable for "John and Mary danced" --> "John danced"; but in the case of appositives it should be treated differently. We obviously didn't mean to model plurals and particularly not in appositives, but in this case the result is a degradation in the system's soundness. I think that due to cases like this it is better to remove the distributive AND than to keep it.
